appinventor/components/src/com/google/appinventor/components/runtime/ListView.java

```diff
@@ -405,5 +412,33 @@ public void TextColor(int argb) {
       textColor = argb;
       setAdapterData();
   }
+  
+  /**
+   * Returns the listview's text font Size
+   *
+   * @return text size as an float
+   */
+  @SimpleProperty(
+      description = "The text size of the listview items.",
+      category = PropertyCategory.APPEARANCE)
+  public int TextSize() {
+    return textSize;
+  }
+
+  /**
+   * Specifies the ListView item's text font size
+   *
+   * @param integer value for font size
+   */
+  @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_NON_NEGATIVE_FLOAT,
+      defaultValue = Component.FONT_DEFAULT_SIZE + "")
+  @SimpleProperty
+  public void TextSize(float fontSize) {
```

Comment:
Will do.  ( I was worried about users overflowing the integer input but I
see that PropertyTypeConstants deals with that. )  I changed this as well
as the default font size to an integer value.  ( Component uses a float
currently)

On Mon Jan 26 2015 at 8:59:42 PM Jeffrey Schiller <notifications@github.com>
wrote:

> In
> appinventor/components/src/com/google/appinventor/components/runtime/ListView.java
> <https://github.com/mit-cml/appinventor-sources/pull/234#discussion_r23582270>
> :
>
> > +  @SimpleProperty(
> > +      description = "The text size of the listview items.",
> > +      category = PropertyCategory.APPEARANCE)
> > +  public int TextSize() {
> > +    return textSize;
> > +  }
> > +
> > +  /**
> > +   * Specifies the ListView item's text font size
> > +   *
> > +   * @param integer value for font size
> > +   */
> > +  @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_NON_NEGATIVE_FLOAT,
> > +      defaultValue = Component.FONT_DEFAULT_SIZE + "")
> > +  @SimpleProperty
> > +  public void TextSize(float fontSize) {
>
> You should probably change this argument to int as well (and update the
> PropertyTypeConstants to integer as well.
>
> -Jeff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-cml/appinventor-sources/pull/234/files#r23582270>.
>